src/main/shell/start-test.sh

```diff
@@ -32,7 +32,7 @@ help() {
   $TestProg -h
 }
 
-if [ $# -lt 1 ] || [ "$1" = -h -o "$1" = --help ]
+if [ $# -lt 2 ] || [ "$1" = -h -o "$1" = --help ]
```

Comment:
I think we should keep the `-lt 1`. The help is shown on no-arg or -h/--help, but we do not really know if 1, 2 or more args are required as this script just forwards to `crdt-partitions.py`.